docker-compose/technical/docker-compose.technical.yml

```diff
@@ -110,3 +110,31 @@ services:
       - PGADMIN_DEFAULT_PASSWORD=admin
     volumes:
       - $PWD/../technical/servers_pgadmin.json:/pgadmin4/servers.json
+
+  prometheus:
+    profiles:
+      - all
+      - metrics
+    image: prom/prometheus
+    command:
+      - --enable-feature=exemplar-storage
+      - --config.file=/etc/prometheus/prometheus.yml
```

Comment:
the [documentation](https://prometheus.io/docs/prometheus/latest/installation/) say its not necessary as it's the default value
```suggestion
```

Reply:
Removed but need to test

Reply:
ah I see...
```
caller=main.go:487 level=error msg="Error loading config (--config.file=prometheus.yml)" file=/prometheus/prometheus.yml err="open prometheus.yml: no such file or directory"
```